### DISCUSSION TITLE
Does adblock prevent ads on BlazorWebView on MAUI

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/aspnetcore#49160] [Q&A]
So if I put advertisement banners in my games, is it possible for the users to use adblock to remove them, if I am using BlazorWebView

**Comment by samsosa** [comment]
No, ordinary advertising is blocked, just like in the browser.
... it depends on the adblock software.

**Reply by Xyncgas** [reply]
So can people install the adblock extension on their pc / mobile and block ad, or I am shipping a new browser that's separate from the one they install, so if I put ad in app the extensions they installed is not running along with my app and blocking it
